### PR TITLE
feat: continuous publish using pkg.pr.new

### PR DESCRIPTION
**Pull request by kongmoumou**
This PR will enable continuous publish wxt packages for every commit & PR on main automatically, instead of publishing it manually using npm command. It would be super easy and fast for testing locally, just update package version pointed to specific commit. It works in my fork, result like this ⬇️

image

All u need to do is [install the Github Application](https://github.com/apps/pkg-pr-new).

Detail see https://github.com/stackblitz-labs/pkg.pr.new

### CODE REVIEW

**Comment by aklinker1** [comment]
Seems intriguing, would simplify releasing test versions. Will review this when I have time!

File: .github/workflows/pkg.pr.new.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          node-version: 20
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Build
+        run: pnpm -r build
+
+      - run: pnpx pkg-pr-new publish './packages/*'
```
Comment (aklinker1) [inline .github/workflows/pkg.pr.new.yml:30]: Lets get this more inline with the rest of the workflows in the repo. Also added `--compact`.

```suggestion
jobs:
  build:
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v4
      - uses: ./.github/actions/setup
      - run: pnpm buildc all
      - run: pnpx pkg-pr-new publish --compact './packages/*'
```

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Alright, this looks good. Cleanup the workflow a bit and we'll merge!

File: .github/workflows/pkg.pr.new.yml (hunk 1 above)
Comment (kongmoumou) [reply .github/workflows/pkg.pr.new.yml]: done~

**Review by aklinker1 — APPROVED** [review]
Thanks for the PR!

**Pull request merged**